frontends/concrete-python/tests/tfhers-utils/src/main.rs

```diff
@@ -337,7 +408,10 @@ fn main() {
 
     match matches.subcommand() {
         Some(("encrypt-with-key", encrypt_matches)) => {
-            let value_str = encrypt_matches.get_one::<String>("value").unwrap();
+            let value_str: Vec<&String> = encrypt_matches
```

Comment:
As we discuss could be nicer to have `--value=1,2,3,4` or anything else to give a tensor in only one argument